DynamoDB client created
Table name table629676317 created
DynamoDB client created
Table name table23267595 created
METHOD: dynamo_batch_execute_statement --- ARGS: []
**AWS** : Success -- Batch execute statement succeeded
**EMULATOR** : Success -- Batch execute statement succeeded
Success -- Table deleted:  table185972671
Success -- Table deleted:  table398534925
Success -- Table deleted:  table480530618
Success -- Table deleted:  table486452545
Success -- Table deleted:  table603693709
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table629676317
Success -- Table deleted:  table864630565
Success -- Table deleted:  table997088336
Success -- Table deleted:  table23267595
DynamoDB client created
Table name table389179505 created
DynamoDB client created
Table name table862765506 created
METHOD: dynamo_batch_get_item --- ARGS: []
**AWS** : Fail -- Batch get item failed; error:  An error occurred (ResourceNotFoundException) when calling the BatchGetItem operation: Requested resource not found
**EMULATOR** : Success -- Batch get item succeeded

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (ResourceNotFoundException) when calling the BatchGetItem operation: Requested resource not found 
EMULATOR: SUCCESS -- Response: 


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table185972671
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table389179505
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table480530618
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table486452545
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table603693709
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table629676317
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table864630565
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table997088336
Success -- Table deleted:  table862765506
DynamoDB client created
Table name table56580188 created
DynamoDB client created
Table name table93133128 created
METHOD: dynamo_batch_write_item --- ARGS: []
**AWS** : Fail -- Batch write item failed; error:  An error occurred (ResourceNotFoundException) when calling the BatchWriteItem operation: Requested resource not found
**EMULATOR** : Success -- Batch write item succeeded

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (ResourceNotFoundException) when calling the BatchWriteItem operation: Requested resource not found 
EMULATOR: SUCCESS -- Response: 


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table389179505
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table486452545
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table56580188
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table603693709
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table629676317
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table864630565
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table997088336
Success -- Table deleted:  table93133128
DynamoDB client created
Table name table699158897 created
DynamoDB client created
Table name table645832291 created
METHOD: dynamo_can_paginate --- ARGS: []
**AWS** : Success -- Can paginate succeeded
**EMULATOR** : Success -- Can paginate succeeded
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table389179505
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table56580188
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table603693709
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table629676317
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table699158897
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table997088336
Success -- Table deleted:  table645832291
DynamoDB client created
Table name table861422651 created
DynamoDB client created
Table name table636928512 created
METHOD: dynamo_close --- ARGS: []
**AWS** : Success -- Close succeeded
**EMULATOR** : Success -- Close succeeded
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table389179505
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table56580188
Success -- Table deleted:  table629676317
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table699158897
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table861422651
Success -- Table deleted:  table636928512
DynamoDB client created
Table name table46012581 created
DynamoDB client created
Table name table610860054 created
METHOD: dynamo_create_backup --- ARGS: []
**AWS** : Fail -- Create backup failed; error:  An error occurred (TableNotFoundException) when calling the CreateBackup operation: Table not found: table46012581
**EMULATOR** : Fail -- Create backup failed; error:  An error occurred (UnknownOperationException) when calling the CreateBackup operation: An unknown operation was requested.

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: Table not found: table46012581 -- Error Code: TableNotFoundException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: An unknown operation was requested. -- Error Code: UnknownOperationException-- HTTP Status code: 400


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table389179505
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table46012581
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table56580188
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table629676317
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table699158897
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table861422651
Success -- Table deleted:  table610860054
DynamoDB client created
Table name table800511216 created
DynamoDB client created
Table name table754476455 created
METHOD: dynamo_create_global_table --- ARGS: []
**AWS** : Fail -- Create global table failed; error:  An error occurred (TableNotFoundException) when calling the CreateGlobalTable operation: Table not found: Table: 'table800511216' not found in region: 'us-east-1'.
**EMULATOR** : Success -- Create global table succeeded

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (TableNotFoundException) when calling the CreateGlobalTable operation: Table not found: Table: 'table800511216' not found in region: 'us-east-1'. 
EMULATOR: SUCCESS -- Response: 


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table389179505
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table46012581
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table56580188
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table629676317
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table699158897
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table800511216
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table861422651
Success -- Table deleted:  table754476455
DynamoDB client created
Table name table673970943 created
DynamoDB client created
Table name table182858538 created
METHOD: dynamo_create_table --- ARGS: []
**AWS** : Success -- Table created
**EMULATOR** : Success -- Table created
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table389179505
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table46012581
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table56580188
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table629676317
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table673970943
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table699158897
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table800511216
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table861422651
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table959098
Success -- Table deleted:  table182858538
Success -- Table deleted:  table4243675
DynamoDB client created
Table name table497152592 created
DynamoDB client created
Table name table162916657 created
METHOD: dynamo_delete_backup --- ARGS: []
**AWS** : Fail -- Delete backup failed; error:  An error occurred (ValidationException) when calling the DeleteBackup operation: Invalid Request: BackupArn is not valid
**EMULATOR** : Fail -- Delete backup failed; error:  An error occurred (UnknownOperationException) when calling the DeleteBackup operation: An unknown operation was requested.

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: Invalid Request: BackupArn is not valid -- Error Code: ValidationException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: An unknown operation was requested. -- Error Code: UnknownOperationException-- HTTP Status code: 400


Success -- Table deleted:  table389179505
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table46012581
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table497152592
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table56580188
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table673970943
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table699158897
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table800511216
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table861422651
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table959098
Success -- Table deleted:  table162916657
DynamoDB client created
Table name table68468757 created
DynamoDB client created
Table name table689066780 created
METHOD: dynamo_delete_item --- ARGS: []
**AWS** : Fail -- Delete item failed; error:  An error occurred (ResourceNotFoundException) when calling the DeleteItem operation: Requested resource not found
**EMULATOR** : Success -- Delete item succeeded

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (ResourceNotFoundException) when calling the DeleteItem operation: Requested resource not found 
EMULATOR: SUCCESS -- Response: 


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table389179505
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table46012581
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table497152592
Success -- Table deleted:  table56580188
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table673970943
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table68468757
Success -- Table deleted:  table699158897
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table800511216
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table861422651
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table959098
Success -- Table deleted:  table689066780
DynamoDB client created
Table name table464470766 created
DynamoDB client created
Table name table91728480 created
METHOD: dynamo_delete_table --- ARGS: []
**AWS** : Fail -- Delete table failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table464470766
**EMULATOR** : Success -- Delete table succeeded

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table464470766 
EMULATOR: SUCCESS -- Response: 


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table389179505
Success -- Table deleted:  table46012581
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table464470766
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table497152592
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table56580188
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table673970943
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table68468757
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table699158897
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table800511216
Success -- Table deleted:  table861422651
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table959098
DynamoDB client created
Table name table379976325 created
DynamoDB client created
Table name table997327820 created
METHOD: dynamo_describe_backup --- ARGS: []
**AWS** : Fail -- Describe backup failed; error:  An error occurred (ValidationException) when calling the DescribeBackup operation: Invalid Request: BackupArn is not valid
**EMULATOR** : Fail -- Describe backup failed; error:  An error occurred (UnknownOperationException) when calling the DescribeBackup operation: An unknown operation was requested.

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: Invalid Request: BackupArn is not valid -- Error Code: ValidationException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: An unknown operation was requested. -- Error Code: UnknownOperationException-- HTTP Status code: 400


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table379976325
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table389179505
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table46012581
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table464470766
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table497152592
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table673970943
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table68468757
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table699158897
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table800511216
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table861422651
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table959098
Success -- Table deleted:  table997327820
DynamoDB client created
Table name table184048054 created
DynamoDB client created
Table name table467876630 created
METHOD: dynamo_describe_continuous_backups --- ARGS: []
**AWS** : Fail -- Describe continuous backups failed; error:  An error occurred (TableNotFoundException) when calling the DescribeContinuousBackups operation: Table not found: table184048054
**EMULATOR** : Success -- Describe continuous backups succeeded

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (TableNotFoundException) when calling the DescribeContinuousBackups operation: Table not found: table184048054 
EMULATOR: SUCCESS -- Response: 


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table184048054
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table379976325
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table46012581
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table464470766
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table497152592
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table673970943
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table68468757
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table800511216
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table959098
Success -- Table deleted:  table467876630
DynamoDB client created
Table name table516917143 created
DynamoDB client created
Table name table709696741 created
METHOD: dynamo_describe_contributor_insights --- ARGS: []
**AWS** : Fail -- Describe contributor insights failed; error:  An error occurred (ResourceNotFoundException) when calling the DescribeContributorInsights operation: Requested resource not found: Table: table516917143 not found
**EMULATOR** : Fail -- Describe contributor insights failed; error:  An error occurred (UnknownOperationException) when calling the DescribeContributorInsights operation: An unknown operation was requested.

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: Requested resource not found: Table: table516917143 not found -- Error Code: ResourceNotFoundException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: An unknown operation was requested. -- Error Code: UnknownOperationException-- HTTP Status code: 400


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table184048054
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table379976325
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table464470766
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table497152592
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table516917143
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table673970943
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table68468757
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table800511216
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table959098
Success -- Table deleted:  table709696741
DynamoDB client created
Table name table910192112 created
DynamoDB client created
Table name table379498351 created
METHOD: dynamo_describe_endpoints --- ARGS: []
**AWS** : Success -- Describe endpoints succeeded
**EMULATOR** : Fail -- Describe endpoints failed; error:  An error occurred (UnknownOperationException) when calling the DescribeEndpoints operation: An unknown operation was requested.

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: SUCCESS -- Response:  
EMULATOR: FAILURE -- Response: An error occurred (UnknownOperationException) when calling the DescribeEndpoints operation: An unknown operation was requested.


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table184048054
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table379976325
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table464470766
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table497152592
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table516917143
Success -- Table deleted:  table673970943
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table68468757
Success -- Table deleted:  table800511216
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table910192112
Success -- Table deleted:  table959098
Success -- Table deleted:  table379498351
DynamoDB client created
Table name table323184513 created
DynamoDB client created
Table name table403484303 created
METHOD: dynamo_describe_exports --- ARGS: []
**AWS** : Fail -- Describe exports failed; error:  'DynamoDB' object has no attribute 'describe_exports'
**EMULATOR** : Fail -- Describe exports failed; error:  'DynamoDB' object has no attribute 'describe_exports'
Exception:  'AttributeError' object has no attribute 'response'
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table184048054
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table323184513
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table379976325
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table464470766
Success -- Table deleted:  table497152592
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table516917143
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table673970943
Success -- Table deleted:  table68468757
Fail -- Table delete failed; error:  An error occurred (ResourceNotFoundException) when calling the DeleteTable operation: Requested resource not found: Table: table800511216 not found
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table910192112
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table959098
Success -- Table deleted:  table403484303
DynamoDB client created
Table name table447488277 created
DynamoDB client created
Table name table992609544 created
METHOD: dynamo_describe_global_table --- ARGS: []
**AWS** : Fail -- Describe global table failed; error:  An error occurred (GlobalTableNotFoundException) when calling the DescribeGlobalTable operation: Global table not found: Global table with name: 'table447488277' does not exist.
**EMULATOR** : Fail -- Describe global table failed; error:  An error occurred (GlobalTableNotFoundException) when calling the DescribeGlobalTable operation: Global table with this name does not exist

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: Global table not found: Global table with name: 'table447488277' does not exist. -- Error Code: GlobalTableNotFoundException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: Global table with this name does not exist -- Error Code: GlobalTableNotFoundException-- HTTP Status code: 400


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table184048054
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table323184513
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table379976325
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table447488277
Success -- Table deleted:  table464470766
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table497152592
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table516917143
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table68468757
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table910192112
Success -- Table deleted:  table992609544
DynamoDB client created
Table name table174004806 created
DynamoDB client created
Table name table506349539 created
METHOD: dynamo_describe_global_table_settings --- ARGS: []
**AWS** : Fail -- Describe global table settings failed; error:  An error occurred (GlobalTableNotFoundException) when calling the DescribeGlobalTableSettings operation: Global table with name: 'table174004806' does not exist.
**EMULATOR** : Fail -- Describe global table settings failed; error:  An error occurred (UnknownOperationException) when calling the DescribeGlobalTableSettings operation: An unknown operation was requested.

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: Global table with name: 'table174004806' does not exist. -- Error Code: GlobalTableNotFoundException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: An unknown operation was requested. -- Error Code: UnknownOperationException-- HTTP Status code: 400


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table174004806
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table184048054
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table323184513
Success -- Table deleted:  table379976325
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table447488277
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table464470766
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table497152592
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table516917143
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table68468757
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table910192112
Success -- Table deleted:  table506349539
DynamoDB client created
Table name table408593852 created
DynamoDB client created
Table name table272962207 created
METHOD: dynamo_describe_import --- ARGS: []
**AWS** : Fail -- Describe import failed; error:  An error occurred (ValidationException) when calling the DescribeImport operation: Invalid Import ARN
**EMULATOR** : Fail -- Describe import failed; error:  An error occurred (UnknownOperationException) when calling the DescribeImport operation: An unknown operation was requested.

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: Invalid Import ARN -- Error Code: ValidationException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: An unknown operation was requested. -- Error Code: UnknownOperationException-- HTTP Status code: 400


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table174004806
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table184048054
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table323184513
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table379976325
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table408593852
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table447488277
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table464470766
Success -- Table deleted:  table516917143
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table910192112
Success -- Table deleted:  table272962207
DynamoDB client created
Table name table704872961 created
DynamoDB client created
Table name table761119046 created
METHOD: dynamo_describe_kinesis_streaming_destination --- ARGS: []
**AWS** : Fail -- Describe kinesis streaming destination failed; error:  An error occurred (ResourceNotFoundException) when calling the DescribeKinesisStreamingDestination operation: Requested resource not found: Table: table704872961 not found
**EMULATOR** : Success -- Describe kinesis streaming destination succeeded

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (ResourceNotFoundException) when calling the DescribeKinesisStreamingDestination operation: Requested resource not found: Table: table704872961 not found 
EMULATOR: SUCCESS -- Response: 


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table174004806
Success -- Table deleted:  table184048054
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table323184513
Fail -- Table delete failed; error:  An error occurred (ResourceNotFoundException) when calling the DeleteTable operation: Requested resource not found: Table: table379976325 not found
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table408593852
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table447488277
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table516917143
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table704872961
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table910192112
Success -- Table deleted:  table761119046
DynamoDB client created
Table name table42520476 created
DynamoDB client created
Table name table495582669 created
METHOD: dynamo_describe_limits --- ARGS: []
**AWS** : Success -- Describe limits succeeded
**EMULATOR** : Success -- Describe limits succeeded
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table174004806
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table184048054
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table323184513
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table408593852
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table42520476
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table447488277
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table516917143
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table704872961
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table910192112
Success -- Table deleted:  table495582669
DynamoDB client created
Table name table475001848 created
DynamoDB client created
Table name table267514342 created
METHOD: dynamo_describe_table --- ARGS: []
**AWS** : Success -- Describe table succeeded
**EMULATOR** : Success -- Describe table succeeded
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table174004806
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table323184513
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table408593852
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table42520476
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table447488277
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table475001848
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table516917143
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table704872961
Success -- Table deleted:  table910192112
Success -- Table deleted:  table267514342
DynamoDB client created
Table name table720296173 created
DynamoDB client created
Table name table667363637 created
METHOD: dynamo_describe_table_replication_auto_scaling --- ARGS: []
**AWS** : Fail -- Describe table replication auto scaling failed; error:  An error occurred (ResourceNotFoundException) when calling the DescribeTableReplicaAutoScaling operation: Global table with name: 'table720296173' does not exist.
**EMULATOR** : Fail -- Describe table replication auto scaling failed; error:  An error occurred (UnknownOperationException) when calling the DescribeTableReplicaAutoScaling operation: An unknown operation was requested.

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: Global table with name: 'table720296173' does not exist. -- Error Code: ResourceNotFoundException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: An unknown operation was requested. -- Error Code: UnknownOperationException-- HTTP Status code: 400


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table174004806
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table323184513
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table408593852
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table42520476
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table447488277
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table475001848
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table704872961
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table720296173
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table910192112
Success -- Table deleted:  table667363637
DynamoDB client created
Table name table509777949 created
DynamoDB client created
Table name table145945611 created
METHOD: dynamo_describe_time_to_live --- ARGS: []
**AWS** : Fail -- Describe time to live failed; error:  An error occurred (ValidationException) when calling the DescribeTimeToLive operation: Cannot describe time to live while table is in CREATING state: Current table state is CREATING
**EMULATOR** : Success -- Describe time to live succeeded

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (ValidationException) when calling the DescribeTimeToLive operation: Cannot describe time to live while table is in CREATING state: Current table state is CREATING 
EMULATOR: SUCCESS -- Response: 


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table174004806
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table323184513
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table408593852
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table42520476
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table447488277
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table475001848
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table509777949
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table704872961
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table720296173
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table910192112
Success -- Table deleted:  table145945611
DynamoDB client created
Table name table952693776 created
DynamoDB client created
Table name table851945619 created
METHOD: dynamo_disable_kinesis_streaming_destination --- ARGS: []
**AWS** : Fail -- Disable kinesis streaming destination failed; error:  An error occurred (ResourceNotFoundException) when calling the DisableKinesisStreamingDestination operation: Requested resource not found: Table: table952693776 not found
**EMULATOR** : Fail -- Disable kinesis streaming destination failed; error:  An error occurred (ValidationException) when calling the DisableKinesisStreamingDestination operation: User does not have a permission to use kinesis stream

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: Requested resource not found: Table: table952693776 not found -- Error Code: ResourceNotFoundException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: User does not have a permission to use kinesis stream -- Error Code: ValidationException-- HTTP Status code: 400


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table174004806
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table323184513
Success -- Table deleted:  table408593852
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table42520476
Success -- Table deleted:  table447488277
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table475001848
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table509777949
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table704872961
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table720296173
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table952693776
Success -- Table deleted:  table851945619
DynamoDB client created
Table name table265055282 created
DynamoDB client created
Table name table895833176 created
METHOD: dynamo_enable_kinesis_streaming_destination --- ARGS: []
**AWS** : Fail -- Enable kinesis streaming destination failed; error:  An error occurred (ResourceNotFoundException) when calling the EnableKinesisStreamingDestination operation: Requested resource not found: Table: table265055282 not found
**EMULATOR** : Fail -- Enable kinesis streaming destination failed; error:  An error occurred (ValidationException) when calling the EnableKinesisStreamingDestination operation: User does not have a permission to use kinesis stream

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: Requested resource not found: Table: table265055282 not found -- Error Code: ResourceNotFoundException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: User does not have a permission to use kinesis stream -- Error Code: ValidationException-- HTTP Status code: 400


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table174004806
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table265055282
Success -- Table deleted:  table323184513
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table408593852
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table42520476
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table447488277
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table475001848
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table509777949
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table704872961
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table720296173
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table952693776
Success -- Table deleted:  table895833176
DynamoDB client created
Table name table559481144 created
DynamoDB client created
Table name table175461377 created
METHOD: dynamo_execute_statement --- ARGS: []
**AWS** : Fail -- Execute statement failed; error:  An error occurred (ResourceNotFoundException) when calling the ExecuteStatement operation: Requested resource not found
**EMULATOR** : Success -- Execute statement succeeded

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (ResourceNotFoundException) when calling the ExecuteStatement operation: Requested resource not found 
EMULATOR: SUCCESS -- Response: 


Success -- Table deleted:  table174004806
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table265055282
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table323184513
Success -- Table deleted:  table42520476
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table447488277
Success -- Table deleted:  table475001848
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table509777949
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table559481144
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table704872961
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table720296173
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table952693776
Success -- Table deleted:  table175461377
DynamoDB client created
Table name table437587036 created
DynamoDB client created
Table name table234020207 created
METHOD: dynamo_execute_transaction --- ARGS: []
**AWS** : Fail -- Execute transaction failed; error:  An error occurred (ValidationException) when calling the ExecuteTransaction operation: Supplied AttributeValue has more than one datatypes set, must contain exactly one of the supported datatypes
**EMULATOR** : Fail -- Execute transaction failed; error:  An error occurred (ValidationException) when calling the ExecuteTransaction operation: Supplied AttributeValue has more than one datatypes set, must contain exactly one of the supported datatypes
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table174004806
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table265055282
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table42520476
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table437587036
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table475001848
Success -- Table deleted:  table509777949
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table559481144
Success -- Table deleted:  table704872961
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table720296173
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table952693776
Success -- Table deleted:  table234020207
DynamoDB client created
Table name table147061355 created
DynamoDB client created
Table name table555793391 created
METHOD: dynamo_export_table_to_point_in_time --- ARGS: []
**AWS** : Fail -- Export table to point in time failed; error:  An error occurred (ValidationException) when calling the ExportTableToPointInTime operation: One or more parameter values were invalid: tableArn is not a valid ARN
**EMULATOR** : Fail -- Export table to point in time failed; error:  An error occurred (UnknownOperationException) when calling the ExportTableToPointInTime operation: An unknown operation was requested.

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: One or more parameter values were invalid: tableArn is not a valid ARN -- Error Code: ValidationException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: An unknown operation was requested. -- Error Code: UnknownOperationException-- HTTP Status code: 400


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table147061355
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table174004806
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table265055282
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table437587036
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table509777949
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table559481144
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table704872961
Success -- Table deleted:  table720296173
Success -- Table deleted:  table952693776
Success -- Table deleted:  table555793391
DynamoDB client created
Table name table307809681 created
DynamoDB client created
Table name table586566803 created
METHOD: dynamo_get_item --- ARGS: []
**AWS** : Fail -- Get item failed; error:  An error occurred (ResourceNotFoundException) when calling the GetItem operation: Requested resource not found
**EMULATOR** : Success -- Get item succeeded

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (ResourceNotFoundException) when calling the GetItem operation: Requested resource not found 
EMULATOR: SUCCESS -- Response: 


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table147061355
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table265055282
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table307809681
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table437587036
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table559481144
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table720296173
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table952693776
Success -- Table deleted:  table586566803
DynamoDB client created
Table name table860204808 created
DynamoDB client created
Table name table483492676 created
METHOD: dynamo_get_paginator --- ARGS: []
**AWS** : Fail -- Get paginator failed; error:  'ListTables'
**EMULATOR** : Fail -- Get paginator failed; error:  'ListTables'
Exception:  'KeyError' object has no attribute 'response'
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table147061355
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table265055282
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table307809681
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table437587036
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table559481144
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table860204808
Fail -- Table delete failed; error:  An error occurred (ResourceNotFoundException) when calling the DeleteTable operation: Requested resource not found: Table: table952693776 not found
Success -- Table deleted:  table483492676
DynamoDB client created
Table name table706877438 created
DynamoDB client created
Table name table982949286 created
METHOD: dynamo_get_waiter --- ARGS: []
**AWS** : Fail -- Get waiter failed; error:  Waiter does not exist: TableExists
**EMULATOR** : Fail -- Get waiter failed; error:  Waiter does not exist: TableExists
Exception:  'ValueError' object has no attribute 'response'
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table147061355
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table265055282
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table307809681
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table437587036
Success -- Table deleted:  table559481144
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table706877438
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table860204808
Success -- Table deleted:  table982949286
DynamoDB client created
Table name table337502121 created
DynamoDB client created
Table name table833510964 created
METHOD: dynamo_import_table --- ARGS: []
**AWS** : Fail -- Import table failed; error:  An error occurred (ValidationException) when calling the ImportTable operation: One or more parameter values were invalid: ReadCapacityUnits and WriteCapacityUnits must both be specified when BillingMode is PROVISIONED
**EMULATOR** : Fail -- Import table failed; error:  An error occurred (UnknownOperationException) when calling the ImportTable operation: An unknown operation was requested.

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: One or more parameter values were invalid: ReadCapacityUnits and WriteCapacityUnits must both be specified when BillingMode is PROVISIONED -- Error Code: ValidationException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: An unknown operation was requested. -- Error Code: UnknownOperationException-- HTTP Status code: 400


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table147061355
Success -- Table deleted:  table265055282
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table307809681
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table337502121
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table437587036
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table559481144
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table706877438
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table860204808
Success -- Table deleted:  table833510964
DynamoDB client created
Table name table739458240 created
DynamoDB client created
Table name table2346365 created
METHOD: dynamo_list_backups --- ARGS: []
**AWS** : Success -- List backups succeeded
**EMULATOR** : Fail -- List backups failed; error:  An error occurred (UnknownOperationException) when calling the ListBackups operation: An unknown operation was requested.

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: SUCCESS -- Response:  
EMULATOR: FAILURE -- Response: An error occurred (UnknownOperationException) when calling the ListBackups operation: An unknown operation was requested.


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table147061355
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table265055282
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table307809681
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table337502121
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table437587036
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table559481144
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table706877438
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table739458240
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table860204808
Success -- Table deleted:  table2346365
DynamoDB client created
Table name table25898694 created
DynamoDB client created
Table name table448736153 created
METHOD: dynamo_list_contributor_insights --- ARGS: []
**AWS** : Fail -- List contributor insights failed; error:  An error occurred (ResourceNotFoundException) when calling the ListContributorInsights operation: Requested resource not found: Table: table25898694 not found
**EMULATOR** : Fail -- List contributor insights failed; error:  An error occurred (UnknownOperationException) when calling the ListContributorInsights operation: An unknown operation was requested.

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: Requested resource not found: Table: table25898694 not found -- Error Code: ResourceNotFoundException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: An unknown operation was requested. -- Error Code: UnknownOperationException-- HTTP Status code: 400


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table147061355
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table25898694
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table265055282
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table307809681
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table337502121
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table437587036
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table706877438
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table739458240
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table860204808
Success -- Table deleted:  table448736153
DynamoDB client created
Table name table355028454 created
DynamoDB client created
Table name table176255664 created
METHOD: dynamo_list_exports --- ARGS: []
**AWS** : Fail -- List exports failed; error:  An error occurred (ValidationException) when calling the ListExports operation: One or more parameter values were invalid: tableArn is not a valid ARN
**EMULATOR** : Fail -- List exports failed; error:  An error occurred (UnknownOperationException) when calling the ListExports operation: An unknown operation was requested.

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: One or more parameter values were invalid: tableArn is not a valid ARN -- Error Code: ValidationException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: An unknown operation was requested. -- Error Code: UnknownOperationException-- HTTP Status code: 400


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table147061355
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table25898694
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table307809681
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table337502121
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table355028454
Success -- Table deleted:  table437587036
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table706877438
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table739458240
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table860204808
Success -- Table deleted:  table176255664
DynamoDB client created
Table name table883357978 created
DynamoDB client created
Table name table924913005 created
METHOD: dynamo_list_global_tables --- ARGS: []
**AWS** : Success -- List global tables succeeded
**EMULATOR** : Success -- List global tables succeeded
Success -- Table deleted:  table147061355
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table25898694
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table307809681
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table337502121
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table355028454
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table437587036
Success -- Table deleted:  table706877438
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table739458240
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table860204808
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table883357978
Success -- Table deleted:  table924913005
DynamoDB client created
Table name table733524843 created
DynamoDB client created
Table name table780424139 created
METHOD: dynamo_list_imports --- ARGS: []
**AWS** : Success -- List imports succeeded
**EMULATOR** : Fail -- List imports failed; error:  An error occurred (UnknownOperationException) when calling the ListImports operation: An unknown operation was requested.

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: SUCCESS -- Response:  
EMULATOR: FAILURE -- Response: An error occurred (UnknownOperationException) when calling the ListImports operation: An unknown operation was requested.


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table147061355
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table25898694
Success -- Table deleted:  table307809681
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table337502121
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table355028454
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table706877438
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table733524843
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table739458240
Success -- Table deleted:  table860204808
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table883357978
Success -- Table deleted:  table780424139
DynamoDB client created
Table name table671849278 created
DynamoDB client created
Table name table384990715 created
METHOD: dynamo_list_tables --- ARGS: []
**AWS** : Success -- List tables succeeded
**EMULATOR** : Success -- List tables succeeded
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table147061355
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table25898694
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table307809681
Success -- Table deleted:  table337502121
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table355028454
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table671849278
Fail -- Table delete failed; error:  An error occurred (ResourceNotFoundException) when calling the DeleteTable operation: Requested resource not found: Table: table706877438 not found
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table733524843
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table739458240
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table860204808
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table883357978
Success -- Table deleted:  table384990715
DynamoDB client created
Table name table651674880 created
DynamoDB client created
Table name table402909598 created
METHOD: dynamo_list_tags_of_resource --- ARGS: []
**AWS** : Fail -- List tags of resource failed; error:  An error occurred (ValidationException) when calling the ListTagsOfResource operation: Invalid TableArn: Invalid ResourceArn provided as input arn:aws:dynamodb:us-east-1:123456789012:table/table651674880
**EMULATOR** : Success -- List tags of resource succeeded

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (ValidationException) when calling the ListTagsOfResource operation: Invalid TableArn: Invalid ResourceArn provided as input arn:aws:dynamodb:us-east-1:123456789012:table/table651674880 
EMULATOR: SUCCESS -- Response: 


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table25898694
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table307809681
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table337502121
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table355028454
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table651674880
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table671849278
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table733524843
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table739458240
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table883357978
Success -- Table deleted:  table402909598
DynamoDB client created
Table name table666209561 created
DynamoDB client created
Table name table953318103 created
METHOD: dynamo_put_item --- ARGS: []
**AWS** : Fail -- Item put failed; error:  An error occurred (ValidationException) when calling the PutItem operation: Supplied AttributeValue is empty, must contain exactly one of the supported datatypes
**EMULATOR** : Fail -- Item put failed; error:  An error occurred (ValidationException) when calling the PutItem operation: Supplied AttributeValue is empty, must contain exactly one of the supported datatypes
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table25898694
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table337502121
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table355028454
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table651674880
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table666209561
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table671849278
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table733524843
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table739458240
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table883357978
Success -- Table deleted:  table953318103
DynamoDB client created
Table name table983120280 created
DynamoDB client created
Table name table869292378 created
METHOD: dynamo_query --- ARGS: []
**AWS** : Fail -- Query failed; error:  An error occurred (ValidationException) when calling the Query operation: Invalid KeyConditionExpression: Syntax error; token: "<EOF>", near: "string"
**EMULATOR** : Fail -- Query failed; error:  An error occurred (ValidationException) when calling the Query operation: Invalid KeyConditionExpression: Syntax error; token: "<EOF>", near: "string"
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table25898694
Success -- Table deleted:  table355028454
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table651674880
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table666209561
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table671849278
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table733524843
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table739458240
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table883357978
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table983120280
Success -- Table deleted:  table869292378
DynamoDB client created
Table name table574082593 created
DynamoDB client created
Table name table355751914 created
METHOD: dynamo_restore_table_from_backup --- ARGS: []
**AWS** : Fail -- Restore table from backup failed; error:  An error occurred (ValidationException) when calling the RestoreTableFromBackup operation: Invalid Request: sseSpecificationOverride must be provided for cross-region restores
**EMULATOR** : Fail -- Restore table from backup failed; error:  An error occurred (UnknownOperationException) when calling the RestoreTableFromBackup operation: An unknown operation was requested.

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: Invalid Request: sseSpecificationOverride must be provided for cross-region restores -- Error Code: ValidationException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: An unknown operation was requested. -- Error Code: UnknownOperationException-- HTTP Status code: 400


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table25898694
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table355028454
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table574082593
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table651674880
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table666209561
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table671849278
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table733524843
Success -- Table deleted:  table739458240
Success -- Table deleted:  table883357978
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table983120280
Success -- Table deleted:  table355751914
DynamoDB client created
Table name table379219891 created
DynamoDB client created
Table name table103393314 created
METHOD: dynamo_restore_table_to_point_in_time --- ARGS: []
Exception:  'module' object is not callable
Success -- Table deleted:  table25898694
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table355028454
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table379219891
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table574082593
Success -- Table deleted:  table651674880
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table666209561
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table671849278
Success -- Table deleted:  table733524843
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table739458240
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table883357978
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table983120280
Success -- Table deleted:  table103393314
DynamoDB client created
Table name table531786006 created
DynamoDB client created
Table name table45748590 created
METHOD: dynamo_scan --- ARGS: []
**AWS** : Fail -- Scan failed; error:  An error occurred (ValidationException) when calling the Scan operation: ExpressionAttributeValues contains invalid value: Supplied AttributeValue is empty, must contain exactly one of the supported datatypes for key :val
**EMULATOR** : Fail -- Scan failed; error:  An error occurred (ValidationException) when calling the Scan operation: ExpressionAttributeValues contains invalid value: Supplied AttributeValue is empty, must contain exactly one of the supported datatypes for key :val
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table25898694
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table379219891
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table531786006
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table574082593
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table651674880
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table666209561
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table671849278
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table733524843
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table883357978
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table983120280
Success -- Table deleted:  table45748590
DynamoDB client created
Table name table812123247 created
DynamoDB client created
Table name table48182315 created
METHOD: dynamo_tag_resource --- ARGS: []
**AWS** : Fail -- Tag resource failed; error:  An error occurred (ValidationException) when calling the TagResource operation: Invalid TableArn: Invalid ResourceArn provided as input arn:aws:dynamodb:us-east-1:123456789012:table/table812123247
**EMULATOR** : Success -- Tag resource succeeded

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (ValidationException) when calling the TagResource operation: Invalid TableArn: Invalid ResourceArn provided as input arn:aws:dynamodb:us-east-1:123456789012:table/table812123247 
EMULATOR: SUCCESS -- Response: 


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table25898694
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table379219891
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table531786006
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table574082593
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table651674880
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table666209561
Success -- Table deleted:  table671849278
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table733524843
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table812123247
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table983120280
Success -- Table deleted:  table48182315
DynamoDB client created
Table name table291363127 created
DynamoDB client created
Table name table301038394 created
METHOD: dynamo_transact_get_items --- ARGS: []
**AWS** : Fail -- Transact get items failed; error:  An error occurred (ValidationException) when calling the TransactGetItems operation: ExpressionAttributeNames contains invalid key: Syntax error; key: "string"
**EMULATOR** : Fail -- Transact get items failed; error:  An error occurred (ValidationException) when calling the TransactGetItems operation: ExpressionAttributeNames contains invalid key: Syntax error; key: "string"
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table291363127
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table379219891
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table531786006
Success -- Table deleted:  table574082593
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table666209561
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table671849278
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table812123247
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table983120280
Success -- Table deleted:  table301038394
DynamoDB client created
Table name table790876347 created
DynamoDB client created
Table name table574544471 created
METHOD: dynamo_transact_write_items --- ARGS: []
**AWS** : Fail -- Transact write items failed; error:  An error occurred (ValidationException) when calling the TransactWriteItems operation: ExpressionAttributeNames contains invalid key: Syntax error; key: "string"
**EMULATOR** : Fail -- Transact write items failed; error:  An error occurred (ValidationException) when calling the TransactWriteItems operation: The provided key element does not match the schema

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: ExpressionAttributeNames contains invalid key: Syntax error; key: "string" -- Error Code: ValidationException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: The provided key element does not match the schema -- Error Code: ValidationException-- HTTP Status code: 400


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table291363127
Success -- Table deleted:  table379219891
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table531786006
Fail -- Table delete failed; error:  An error occurred (ResourceNotFoundException) when calling the DeleteTable operation: Requested resource not found: Table: table574082593 not found
Success -- Table deleted:  table666209561
Fail -- Table delete failed; error:  An error occurred (ResourceNotFoundException) when calling the DeleteTable operation: Requested resource not found: Table: table671849278 not found
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table790876347
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table812123247
Success -- Table deleted:  table983120280
Success -- Table deleted:  table574544471
DynamoDB client created
Table name table858126693 created
DynamoDB client created
Table name table486452545 created
METHOD: dynamo_untag_resource --- ARGS: []
**AWS** : Fail -- Untag resource failed; error:  An error occurred (ValidationException) when calling the UntagResource operation: Invalid TableArn: Invalid ResourceArn provided as input arn:aws:dynamodb:us-east-1:123456789012:table/table858126693
**EMULATOR** : Success -- Untag resource succeeded

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (ValidationException) when calling the UntagResource operation: Invalid TableArn: Invalid ResourceArn provided as input arn:aws:dynamodb:us-east-1:123456789012:table/table858126693 
EMULATOR: SUCCESS -- Response: 


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table291363127
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table379219891
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table531786006
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table666209561
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table790876347
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table812123247
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table858126693
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table983120280
Success -- Table deleted:  table486452545
DynamoDB client created
Table name table387667858 created
DynamoDB client created
Table name table185972671 created
METHOD: dynamo_update_continuous_backups --- ARGS: []
**AWS** : Fail -- Update continuous backups failed; error:  An error occurred (TableNotFoundException) when calling the UpdateContinuousBackups operation: Table not found: table387667858
**EMULATOR** : Success -- Update continuous backups succeeded

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (TableNotFoundException) when calling the UpdateContinuousBackups operation: Table not found: table387667858 
EMULATOR: SUCCESS -- Response: 


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table291363127
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table379219891
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table387667858
Success -- Table deleted:  table531786006
Fail -- Table delete failed; error:  An error occurred (ResourceNotFoundException) when calling the DeleteTable operation: Requested resource not found: Table: table666209561 not found
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table790876347
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table812123247
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table858126693
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table983120280
Success -- Table deleted:  table185972671
DynamoDB client created
Table name table819752192 created
DynamoDB client created
Table name table864630565 created
METHOD: dynamo_update_contributor_insights --- ARGS: []
**AWS** : Fail -- Update contributor insights failed; error:  An error occurred (ResourceNotFoundException) when calling the UpdateContributorInsights operation: Requested resource not found: Table: table819752192 not found
**EMULATOR** : Fail -- Update contributor insights failed; error:  An error occurred (UnknownOperationException) when calling the UpdateContributorInsights operation: An unknown operation was requested.

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: Requested resource not found: Table: table819752192 not found -- Error Code: ResourceNotFoundException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: An unknown operation was requested. -- Error Code: UnknownOperationException-- HTTP Status code: 400


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table291363127
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table387667858
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table531786006
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table790876347
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table812123247
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table819752192
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table858126693
Fail -- Table delete failed; error:  An error occurred (ResourceNotFoundException) when calling the DeleteTable operation: Requested resource not found: Table: table983120280 not found
Success -- Table deleted:  table864630565
DynamoDB client created
Table name table87776406 created
DynamoDB client created
Table name table997088336 created
METHOD: dynamo_update_global_table --- ARGS: []
**AWS** : Fail -- Update global table failed; error:  An error occurred (GlobalTableNotFoundException) when calling the UpdateGlobalTable operation: Global table not found: Global table with name: 'table87776406' does not exist.
**EMULATOR** : Fail -- Update global table failed; error:  An error occurred (GlobalTableNotFoundException) when calling the UpdateGlobalTable operation: Global table with this name does not exist

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: Global table not found: Global table with name: 'table87776406' does not exist. -- Error Code: GlobalTableNotFoundException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: Global table with this name does not exist -- Error Code: GlobalTableNotFoundException-- HTTP Status code: 400


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table291363127
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table387667858
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table531786006
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table790876347
Success -- Table deleted:  table812123247
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table819752192
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table858126693
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table87776406
Success -- Table deleted:  table997088336
DynamoDB client created
Table name table953135675 created
DynamoDB client created
Table name table603693709 created
METHOD: dynamo_update_global_table_settings --- ARGS: []
**AWS** : Fail -- Update global table settings failed; error:  An error occurred (ValidationException) when calling the UpdateGlobalTableSettings operation: One or more parameter values were invalid: Failed to update settings for global table with name ‘table953135675’: Parameters 'MaximumUnits', 'ScalingPolicyUpdate', 'MinimumUnits' must be left blank when disabling auto scaling.
**EMULATOR** : Fail -- Update global table settings failed; error:  An error occurred (UnknownOperationException) when calling the UpdateGlobalTableSettings operation: An unknown operation was requested.

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: One or more parameter values were invalid: Failed to update settings for global table with name ‘table953135675’: Parameters 'MaximumUnits', 'ScalingPolicyUpdate', 'MinimumUnits' must be left blank when disabling auto scaling. -- Error Code: ValidationException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: An unknown operation was requested. -- Error Code: UnknownOperationException-- HTTP Status code: 400


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table291363127
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table387667858
Success -- Table deleted:  table790876347
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table812123247
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table819752192
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table858126693
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table87776406
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table953135675
Success -- Table deleted:  table603693709
DynamoDB client created
Table name table772497438 created
DynamoDB client created
Table name table480530618 created
METHOD: dynamo_update_item --- ARGS: []
**AWS** : Fail -- Item update failed; error:  An error occurred (ResourceNotFoundException) when calling the UpdateItem operation: Requested resource not found
**EMULATOR** : Success -- Item updated

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (ResourceNotFoundException) when calling the UpdateItem operation: Requested resource not found 
EMULATOR: SUCCESS -- Response: 


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table291363127
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table387667858
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table772497438
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table790876347
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table812123247
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table819752192
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table858126693
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table87776406
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table953135675
Success -- Table deleted:  table480530618
DynamoDB client created
Table name table91023259 created
DynamoDB client created
Table name table398534925 created
METHOD: dynamo_update_replica_autoscaling --- ARGS: []
**AWS** : Fail -- Replica autoscaling update failed; error:  An error occurred (ValidationException) when calling the UpdateTableReplicaAutoScaling operation: Failed to update settings for global table with name: ‘table91023259’: Parameters 'MaximumUnits', 'MinimumUnits' are required unless auto scaling is being disabled.
**EMULATOR** : Fail -- Replica autoscaling update failed; error:  An error occurred (UnknownOperationException) when calling the UpdateTableReplicaAutoScaling operation: An unknown operation was requested.

DISCREPANCY FOUND!

--Error message mismatch--
CLOUD: FAILURE  -- Error Message: Failed to update settings for global table with name: ‘table91023259’: Parameters 'MaximumUnits', 'MinimumUnits' are required unless auto scaling is being disabled. -- Error Code: ValidationException -- HTTP Status code: 400
EMULATOR: FAILURE -- Error Message: An unknown operation was requested. -- Error Code: UnknownOperationException-- HTTP Status code: 400


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table291363127
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table387667858
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table772497438
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table790876347
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table819752192
Success -- Table deleted:  table858126693
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table87776406
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table91023259
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table953135675
Success -- Table deleted:  table398534925
DynamoDB client created
Table name table720347714 created
DynamoDB client created
Table name table886242903 created
METHOD: dynamo_update_table --- ARGS: []
**AWS** : Fail -- Table update failed; error:  An error occurred (ResourceInUseException) when calling the UpdateTable operation: Attempt to change a resource which is still in use: Table is being created: table720347714
**EMULATOR** : Success -- Table updated

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (ResourceInUseException) when calling the UpdateTable operation: Attempt to change a resource which is still in use: Table is being created: table720347714 
EMULATOR: SUCCESS -- Response: 


Success -- Table deleted:  table291363127
Success -- Table deleted:  table387667858
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table720347714
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table772497438
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table819752192
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table858126693
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table87776406
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table91023259
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table953135675
Success -- Table deleted:  table886242903
DynamoDB client created
Table name table214061542 created
DynamoDB client created
Table name table347648150 created
METHOD: dynamo_update_ttl --- ARGS: []
**AWS** : Fail -- TTL update failed; error:  An error occurred (ResourceNotFoundException) when calling the UpdateTimeToLive operation: Requested resource not found: Table: table214061542 not found
**EMULATOR** : Success -- TTL updated

DISCREPANCY FOUND!

--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (ResourceNotFoundException) when calling the UpdateTimeToLive operation: Requested resource not found: Table: table214061542 not found 
EMULATOR: SUCCESS -- Response: 


Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table214061542
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table291363127
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table387667858
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table720347714
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table772497438
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table819752192
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being deleted: table858126693
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table87776406
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table91023259
Fail -- Table delete failed; error:  An error occurred (ResourceInUseException) when calling the DeleteTable operation: Attempt to change a resource which is still in use: Table is being created: table953135675
Success -- Table deleted:  table347648150